src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star stars__star--notactive"></li>
```

Comment:
```suggestion
      <li class="stars__star"></li>
```
remove this class everywhere. you need to do it using css